src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StandardMutectAnnotations.java

```diff
@@ -1,7 +1,7 @@
 package org.broadinstitute.hellbender.tools.walkers.annotator;
 
 /**
- * This is a marker interface used to indicate which annotations are "Standard" for MuTect only.
+ * This is a marker interface used to indicate which annotations are "Standard" for Mutect2 only.
  */
-public interface StandardSomaticAnnotation {
+public interface StandardMutectAnnotations extends Annotation {
```

Comment:
Rename to `StandardMutectAnnotation` (without the plural), for grammatical consistency with the other annotation groups.

Reply:
done